airbyte-integrations/connectors/source-mode/manifest.yaml

```diff
@@ -0,0 +1,6255 @@
+                stream:
+                  $ref: "#/definitions/streams/reports"
+                parent_key: token
+                partition_field: report_token
+          - type: SubstreamPartitionRouter
+            parent_stream_configs:
+              - type: ParentStreamConfig
+                stream:
+                  $ref: "#/definitions/streams/queries"
+                parent_key: token
+                partition_field: query_token
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/charts"
+    groups:
+      type: DeclarativeStream
+      name: groups
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: /{{ config["workspace"] }}/groups
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - _embedded
+              - groups
+      primary_key:
+        - token
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/groups"
+    spaces:
+      type: DeclarativeStream
+      name: spaces
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: /{{ config["workspace"] }}/spaces
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - _embedded
+              - spaces
+      primary_key:
+        - id
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/spaces"
+    queries:
+      type: DeclarativeStream
+      name: queries
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: >-
+            /{{ config["workspace"] }}/reports/{{ stream_partition.report_token
+            }}/queries
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - _embedded
+              - queries
+        partition_router:
+          type: SubstreamPartitionRouter
+          parent_stream_configs:
+            - type: ParentStreamConfig
+              stream:
+                $ref: "#/definitions/streams/reports"
+              parent_key: token
+              partition_field: report_token
+      primary_key:
+        - id
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/queries"
+    reports:
+      type: DeclarativeStream
+      name: reports
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: >-
+            /{{ config['workspace'] }}/spaces/{{ stream_partition.space_token
```

Comment:
Nice implementation! I'm curious to understand if using this https://mode.com/developer/api-reference/analytics/reports/#listReportsUsingDataSource endpoint have the same output. From an use case view it can be more helpful to have reports linked to their data source.